services/kubetunnel/0.0.15/defaults/cm.yaml

```diff
@@ -16,3 +16,17 @@ data:
       kubectlImage:
         repository: bitnami/kubectl
         tag: 1.24.6
+    deploymentAnnotations:
+      secret.reloader.stakater.com/reload: "kubetunnel-reverseproxy-tls"
+    ingress:
```

Comment:
Can we enforce the mTLS setup in this configuration to prevent other components from using the reverse-proxy without passing through the authenticated ingress?